app/javascript/app/components/sectors-agriculture/drivers-of-emissions/historical-emissions-graph/historical-emissions-graph-selectors/pie-chart-selectors.js

```diff
@@ -8,6 +8,7 @@ import { GREY_CHART_COLORS } from 'data/constants';
 import { format } from 'd3-format';
 
 const AGRICULTURE_COLOR = '#0677B3';
+const LUCF_SECTORS = ['Total excluding LUCF', 'Total including LUCF'];
```

Comment:
for naming accuracy I'd call these "AGGREGATED_SECTORS" as they refer to sum of the other sectors, hence the double counting.